pallets/balances/src/lib.rs

```diff
@@ -508,6 +508,11 @@ impl<T: Trait> Module<T> {
         // Commented out for now - but consider it instructive.
         // assert!(!Self::total_balance(who).is_zero());
         <FreeBalance<T>>::insert(who, balance);
+
+        if balance <= 0u128.into() {
```

Comment:
Can you say more generally here, `Zero::zero()`?

Reply:
Yeah it can be